regression/bitwuzla/memset-good/memset-good.c

```diff
@@ -1,5 +1,5 @@
-typedef unsigned int __kernel_size_t;
-typedef __kernel_size_t size_t;
+#include <string.h>
```

Comment:
Do we really need to modify this regression test?

Reply:
Removed the commit that changes the test case.